.github/workflows/ci.yml

```
@@ -266,6 +265,11 @@ jobs:
         PGP_PASSWORD: ${{ secrets.PGP_PASSPHRASE }}
         SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
         SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+    - name: Update stable branch
+      uses: ad-m/github-push-action@master
```

Not sure about using `master` for GitHub actions. I'd rather use a tag or a commit, so that we can quickly review what's new before bumping it (dependabot usually links the diff in its update PR).
```suggestion
      uses: ad-m/github-push-action@8407731efefc0d8f72af254c74276b7a90be36e1
```